pkg/tbtc/signing.go

```diff
@@ -39,16 +36,12 @@ func newSigningRetryLoop(
 	signingGroupOperators chain.Addresses,
 	chainConfig *ChainConfig,
 ) *signingRetryLoop {
-	// Pre-compute the 8-byte seed that is needed for the random
-	// retry algorithm. Since the original message passed as parameter
-	// can have a variable length, it is safer to take the first 8 bytes
-	// of sha256(seed) as the randomRetrySeed.
+	// Compute the 8-byte seed needed for the random retry algorithm. We take
+	// the first 8 bytes of the hash of the signed message. This allows us to
+	// not care in this piece of the code about the length of the message and
+	// how this message is proposed.
```

Comment:
Can we do similar comment update in `pkg/tbtc/dkg.go`?

Reply:
See 916f3173